meetups/2019/02/ivo-lukac-edi-modric--open-source-to-the-core

```diff
@@ -0,0 +1,21 @@
+# Open source to the core
+
+- __Speaker__: Ivo Lukač & Edi Modrić
+- __Duration__: 25 min
+
+## Description
+
+After many years of opensourcing lot of smaller features in eZ and Symfony space, we now opensourced
+our flagship product: Netgen Layouts. In this talk we will share why and how you can start using it.
```

Comment:
Sorry guys, but title and description don't really go together... Title is about opensourcing while description is about how to use your product...